src/pydicom/pixels/processing.py

```diff
@@ -974,20 +1081,52 @@ def _convert_YBR_FULL_to_RGB(arr: "np.ndarray") -> "np.ndarray":
     """
     orig_dtype = arr.dtype
 
-    ybr_to_rgb = np.asarray(
-        [
-            [1.000, 1.000, 1.000],
-            [0.000, -0.114 * 1.772 / 0.587, 1.772],
-            [1.402, -0.299 * 1.402 / 0.587, 0.000],
-        ],
-        dtype=np.float32,
-    )
+    arr = arr.astype(np.float32)
+    arr -= [0, 2 ** (bit_depth - 1), 2 ** (bit_depth - 1)]
```

Comment:
What is this doing? Maybe a comment would help...

Reply:
For 8-bit RGB(0, 0, 0) is YCbCr(0, 128, 128) - a completely black pixel is the "same" as 0 luminance and half-range red and blue chrominance. When converting from RGB -> YCbCr there is a final leveling step that adds [0, 128, 128] (for 8-bit), so in a YCbCr -> RGB conversion, as above, the first step is to remove that leveling.

This is just a slightly modified version of T.871 to account for N-bit depth.

The two YBR_PARTIAL colour spaces use [16, 128, 128] as the value for a completely black pixel.

Reply:
YCbCr -> RGB (my highlighting)
![image](https://github.com/user-attachments/assets/33b0b7f5-7278-4e1e-8ec6-1acf2c905a50)

Reply:
Thanks!